hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/HermesMetrics.java

```diff
@@ -25,6 +25,7 @@
 public class HermesMetrics {
 
     public static final String REPLACEMENT_CHAR = "_";
+    public static final String TMP_REPLACEMENT_CHAR = "-x-";
```

Comment:
If someone have a better temporary replacement char - feel free to suggest

Reply:
`-x-` is a chars combination that passes subscription name regexp: `[a-zA-Z0-9_.-]+`, shouldn't we use something that can't be provided by users, i.e. use special chars like `|`, `=`... ?

Reply:
does this mean that we are going to have names like `my-subscripti-x-on` in graphite? I would rather not, as this introduces some magic that our customers need to be aware of if they want to see metrics from they subscriptions

Reply:
Sorry, have totally missed you comment, Daniel. I've tried to use symbol `€` but somewhere in internals (Graphite url encoding?) it breaks the behaviour. So decided to ask for an another pair of eyes. 
Ok, will rework.

Reply:
*bump* are we going to have metrics published under names like `my€subscription` in Graphite? (or any other char), what does `tmp` mean in this context?

Reply:
`tmp` for `temporary`, probably `UNDERSCORE_REPLACEMENT_CHAR` should be better. Yes, metrics will be under another name. Objections?

Reply:
Yes, strong objections :) This means that all clients need to be aware of this transformation. Also we break existing behaviour - what if someone has alert triggered by this metric? We will break it for him.

Why do we have to make this change in Hermes?